prow/external-plugins/cherrypicker/server.go

```diff
@@ -556,7 +556,7 @@ func (s *Server) ensureForkExists(org, repo string) (string, error) {
 	fork := s.botUser.Login + "/" + repo
 
 	// fork repo if it doesn't exsit
-	if _, err := s.ghc.EnsureFork(s.botUser.Login, org, repo); err != nil {
+	if repo, err := s.ghc.EnsureFork(s.botUser.Login, org, repo); err != nil {
```

Comment:
The only case in which this code will use the name from `EnsureFork` is if there was an error in which case we don't do anything anyways, this doesn't address Steves comment at all

Reply:
I suggest you write some tests for this method to be confident your changes do what you think they do

Reply:
Done. Thanks for the feedback